src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from "react";
+import propTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import DOMPurify from "dompurify";
+import styles from "components/task/create-new/index.module.scss";
+import Button from "components/base/button";
+import TextArea from "components/base/text-area";
+import Image from "components/base/image";
+import { addTaskToTodo } from "store/actions/todo";
+import {
+  ALT_TAG_ICON_DELETE,
+  ATTRIBUTE_TITLE,
+  ERROR_MESSAGE_TASK_TITLE,
+  ICON_DELETE,
+  TYPE_BUTTON,
+} from "utils/const";
+import { generateTaskObject, parseForm } from "utils/helper";
+
+function CreateTask({ onSuccessfullTaskEntry }) {
+  const dispatch = useDispatch();
+  const taskInputRef = useRef(null);
+  const [errorMsg, setErrorMsg] = useState("");
+
+  function handleAddTaskSubmit(event) {
+    event.preventDefault();
+    const formData = parseForm(event.target);
+    const sanitizedTitle = DOMPurify.sanitize(formData.title);
+    if (sanitizedTitle === "") {
+      setErrorMsg(ERROR_MESSAGE_TASK_TITLE);
+      taskInputRef.current.focus();
+      return;
+    }
+    dispatch(addTaskToTodo(generateTaskObject(sanitizedTitle)));
+    taskInputRef.current.value = "";
+    onSuccessfullTaskEntry();
+  }
+
+  useEffect(() => {
+    taskInputRef.current.focus();
+  }, []);
+
+  return (
+    <form onSubmit={handleAddTaskSubmit}>
+      <TextArea ref={taskInputRef} name={ATTRIBUTE_TITLE} errorMsg={errorMsg} />
+      <div className={`flex items-center ${styles.buttonContainer}`}>
+        <Button className={styles.addTaskBtn}>Add Task</Button>
+        <Button buttonType={TYPE_BUTTON}>
+          <Image src={ICON_DELETE} alt={ALT_TAG_ICON_DELETE} />
+        </Button>
```

Comment:
is it part of adding task?

Reply:
This button's functionality is to delete task. In the next [PR](https://github.com/azo-wtag/todo-app-react/pull/2) I have worked on the functionality of this button. Thank you